platform/android/java/lib/src/org/godotengine/godot/xr/regular/RegularConfigChooser.java

```diff
@@ -52,13 +52,13 @@ public class RegularConfigChooser implements GLSurfaceView.EGLConfigChooser {
 	 * perform actual matching in chooseConfig() below.
 	 */
 	private static int EGL_OPENGL_ES2_BIT = 4;
-	private static int[] s_configAttribs2 = {
+	private static int[] s_configAttribs3 = {
 		EGL10.EGL_RED_SIZE, 4,
 		EGL10.EGL_GREEN_SIZE, 4,
 		EGL10.EGL_BLUE_SIZE, 4,
-		//  EGL10.EGL_DEPTH_SIZE,     16,
-		// EGL10.EGL_STENCIL_SIZE,   EGL10.EGL_DONT_CARE,
-		EGL10.EGL_RENDERABLE_TYPE, EGL_OPENGL_ES2_BIT,
+		// EGL10.EGL_DEPTH_SIZE,     16,
+		//  EGL10.EGL_STENCIL_SIZE,   EGL10.EGL_DONT_CARE,
```

Comment:
Nit but the diff on those two lines seems avoidable (though we could take the opportunity and indeed remove that extra whitespace).